alcs-frontend/src/app/features/application/lfng-info/lfng-info.component.html

```diff
@@ -108,35 +140,75 @@ <h4>Plans & Bylaws: Zoning</h4>
 
     <h4>Resolution</h4>
     <div class="review-table">
-      <div class="full-row">
+      <div
+        class="full-row"
+        *ngIf="
+          !isFirstNationGovernment &&
+          (applicationReview.isOCPDesignation === null || applicationReview.isSubjectToZoning === null)
+        "
+      >
+        Please complete both Step 2 Plans & Bylaws: OCP and Step 3 Plans & Bylaws: Zoning to continue with this step.
+      </div>
+
+      <div
+        class="full-row"
+        *ngIf="
+          !isFirstNationGovernment &&
+          applicationReview.isOCPDesignation === false &&
+          applicationReview.isSubjectToZoning === false
+        "
+      >
+        By indicating that the parcel(s) is not subject to Local Government OCP or Zoning,
+        <strong>no authorizing resolution is required</strong> as per S. 25 (3) or S. 29 (4) of the <em>ALC Act</em>.
+        The only option available is to forward this application on to the ALC.
+      </div>
+
+      <div
+        class="full-row"
+        *ngIf="
+          isFirstNationGovernment ||
```

Comment:
This is getting pretty complex, can we move this to the component and use a cleaner boolean here?

Reply:
Updated!